examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -57,7 +57,7 @@ constexpr size_t kOtaHeaderMaxSize   = 1024;
 // Arbitrary BDX Transfer Params
 constexpr uint32_t kMaxBdxBlockSize                 = 1024;
 constexpr chip::System::Clock::Timeout kBdxTimeout  = chip::System::Clock::Seconds16(5 * 60); // OTA Spec mandates >= 5 minutes
-constexpr chip::System::Clock::Timeout kBdxPollFreq = chip::System::Clock::Milliseconds32(500);
+constexpr uint32_t kBdxPollFrequency                = 500; // poll every 500ms by default
```

Comment:
This is not a frequency, it's an interval. Please rename to `kBdxServerPollIntervalMillis`

Reply:
> This is not a frequency, it's an interval. Please rename to `kBdxServerPollIntervalMillis`.
>
> Also, since this is just an _example_, and it's a single process, single client version, we can move this to 50ms by default and just make it faster period...

Done.